ui/src/components/flows/FlowRun.vue

```diff
@@ -13,6 +13,7 @@
                 :prop="input.name"
             >
                 <el-input
+                    @keydown.enter.prevent
```

Comment:
Ugly hack due to VueJS bug where submit is triggered  while pressing enter when focus is on input.